bindings/python/Makefile

```diff
@@ -1,10 +1,13 @@
 default: build
 
-
-edlib: ../../edlib
-	cp -R ../../edlib .
+.PHONY:
+edlib:
+	# copy edlib from above if not already present
+	# (in travisCI we will have already copied it
+	[ -d "edlib" ] && echo "Directory 'edlib' exists." || cp -R ../../edlib .
 
 pyedlib.bycython.cpp: edlib.pyx cedlib.pxd
+	python -m pip install cython
```

Comment:
Cool idea! Could you also update README so it does not mention cython and cogapp as dependencies any more?

Reply:
Done